packages/models/package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@ckb-lumos/base": "0.19.0",
     "@ckb-lumos/codec": "0.19.0",
+    "@ckb-lumos/experiment-tx-assembler": "^0.19.0",
```

Comment:
> Dependency should be pinned

Do you mean to add the modification to the description of this PR?

Reply:
The default behavior of `npm` is to use a range version as `^0.19.0`, in some cases, the `lock` will not work(e.g. `yarn.lock` is not recognized by npm before), and the patch updates as `0.19.x` will be installed locally. It would be a bit dangerous since `0.19.x` is not reviewed. So pinned version is suggested

Ref: https://docs.renovatebot.com/dependency-pinning/